### DISCUSSION TITLE
Github 500 error auth

### DISCUSSION THREAD

**Discussion by Snipeur060** [Labs64/NetLicensing-Community#122] [Ideas & Feature Requests]
**Issue description**
You can't use github account to register / login

**How to reproduce**
1. Register
2. Use github method
3. Wait and 500 error appear

**Expected behavior**
Login and register

**Screenshots**
(no screenshot)

**Additional context**

**Comment by v-rudkovskiy** [comment]
PR: https://github.com/Labs64/NetLicensing-Core/pull/114

**Comment by r-brown** [comment]
Hello @Snipeur060 please make sure your email address specified at [GitHub settings](https://github.com/settings/emails) is set as "public"
e.g.
_"Keep my email addresses"_ private flag is unset
